release-1.8/release_notes_draft.md

```diff
@@ -102,6 +105,11 @@ kind.
  Users that have any code that depends on mutable selectors may continue to use 
  the apps/v1beta1 API for this release, but they should begin migration to code 
  that does depend on mutable selectors.
+ - Extended resources outside the `*kubernetes.io/` namespace are treated as
```

Comment:
Extended Resources vs Extended resources:  keep it consistent across the document.